cdk/l3-cdk-resources/encryption-at-rest-express/test/index.test.ts

```diff
@@ -0,0 +1,108 @@
+import { App, Stack } from 'aws-cdk-lib';
+import { Template } from 'aws-cdk-lib/assertions';
+import { AtlasEncryptionAtRestExpress } from '../src';
+
+
+const PROJECT_ID = 'PROJ_ID';
+const PROJECT_NAME = 'TEST';
+const ROLE_ID = 'ROLE_ID';
+const CUSTOMER_MASTER_KEY = 'MASTER_KEY';
+const DATABASE_NAME = 'DATABASE_NAME';
+const PASSWORD = 'PASSWORD';
+const DATABASE_USER_NAME = 'atlas-user';
+const RESOURCE_NAME_CLUSTER = 'MongoDB::Atlas::Cluster';
+const RESOURCE_NAME_DB_USER = 'MongoDB::Atlas::DatabaseUser';
+const RESOURCE_ENCRYPTION_AT_REST = 'MongoDB::Atlas::EncryptionAtRest';
+const RESOURCE_ACCESS_LIST = 'MongoDB::Atlas::ProjectIpAccessList';
+const REGION = 'US_EAST_1';
+const INSTANCE_SIZE = 'M30';
+const ADMIN_DB= 'admin';
+const ROLE_NAME = 'atlasAdmin';
+const EBS_VOLUME_TYPE = 'STANDARD';
+const IP_ADDRESS = '190.0.0.1';
```

Comment:
same with this one, is it possible to keep IP address closed by default?

Reply:
Same as above, this is a test file and it used only to validate that the resource is correctly created